test/OpenTelemetry.Exporter.Geneva.Tests/UnixUserEventsDataTransportTests.cs

```diff
@@ -143,6 +144,106 @@ public void UserEvents_Disabled_Success_Linux()
         }
     }
 
+    [Fact(Skip = "This would fail on Ubuntu. Skipping for now. See issue: #2326.")]
+    public void UserEvents_Logs_Success_Linux()
+    {
+        EnsureUserEventsEnabled();
+
+        var listener = new PerfTracepointListener(
+            "MicrosoftOpenTelemetryLogs_L4K1",
+            MetricUnixUserEventsDataTransport.MetricsTracepointNameArgs);
+
+        var logsTracepoint = UnixUserEventsDataTransport.Instance.RegisterUserEventProviderForLogs();
+
+        if (listener.IsEnabled())
```

Comment:
Why need fail on this? The tracepoint could be enabled by another process? If so, I think user_events does support multiple readers on the same tracepoint.

Reply:
Removed them. Not really necessary.